Sources/Navigation/DistributedNavigationDelegate.swift

```diff
@@ -416,10 +416,12 @@ extension DistributedNavigationDelegate: WKNavigationDelegate {
         if let expectedNavigation = navigationExpectedToStart, wkNavigation != nil || expectedNavigation.navigationAction.navigationType == .sessionRestoration {
             // regular flow: start .expected navigation
             navigation = expectedNavigation
-        } else {
+        } else if webView.url?.isEmpty == false {
```

Comment:
While it fixes calling `window.open(“”)`, it doesn’t seem to handle `window.open(“about:settings”)` which is used in Duck Player to open settings and as such causes an assertion on debug builds.

Reply:
fixed